### PR TITLE
Update supported web browsers

### PR DESCRIPTION
**Pull request by skjnldsv**
Internet explorer is not supported, Edge have issues so we do not recommend it.

### CODE REVIEW

File: user_manual/webinterface.rst [hunk 1]
```diff
@@ -14,29 +14,29 @@ Web browser requirements
...
 .. No need to translate
 
-* Apple **Safari**
+* Microsoft **Edge**
 
 .. note:: If you want to use Nextcloud Talk you need to run Mozilla **Firefox** 52+
    or Google **Chrome**/Chromium 49+ to have the full experience with video calls and 
    screensharing. Google **Chrome**/Chromium requires a additional plugin for screensharing.
+   
+.. note:: While we try our best to support it, we do **NOT** recommend the usage of microsoft Edge.
```
Comment (szaimen) [inline user_manual/webinterface.rst:38]: are you sure? Microsoft switched to chromium as base for microsoft edge a while ago...

**Review by szaimen — APPROVED** [review]

File: user_manual/webinterface.rst (hunk 1 above)
Comment (skjnldsv) [reply user_manual/webinterface.rst:38]: Yeah, even with that I think we had reports iirc. 
But also, most of us are actually never testing with that browser, could be a safe call to not recommend it.

Comment (skjnldsv) [reply user_manual/webinterface.rst:38]: cc @juliushaertl @danxuliu @ChristophWurst @PVince81 @nickvergessen

Comment (nickvergessen) [reply user_manual/webinterface.rst:38]: Edge is supported and should work flawlessly (apart from a minor issue in Talk)
I still don't recommend using it as it means you run windows, but should be fine in general

If you still keep this as least fix the typo:
```suggestion
.. note:: While we try our best to support it, we do **NOT** recommend the usage of Microsoft Edge.
```

Comment (szaimen) [reply user_manual/webinterface.rst:38]: > I still don't recommend using it as it means you run windows

It also runs on macos, ios android and partially on Linux, if I remeber correctly: https://docs.microsoft.com/en-us/deployedge/microsoft-edge-supported-operating-systems

Comment (nickvergessen) [reply user_manual/webinterface.rst:38]: Thanks for ruining the joke :P

Comment (szaimen) [reply user_manual/webinterface.rst:38]: Sorry 😅

**Pull request merged**